### PR TITLE
Mandana

### PR DESCRIPTION
**Pull request by mandana-mazaheri**
to add study registration page in the main branch

### CODE REVIEW

File: frontend/yarn.lock [hunk 1]
```diff
@@ -2,1126 +2,1162 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.10.4", "@babel/code-frame@^7.12.13", "@babel/code-frame@^7.5.5":
```
Comment (shiyuchen6688) [inline frontend/yarn.lock:5]: Are you using yarn as your package manager? we probably should not use npm and yarn at the same time, not sure if mixing those two will cause us problems

File: frontend/src/components/NavigationBar.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { Navbar, Nav, Container } from 'react-bootstrap'
```
Comment (shiyuchen6688) [inline frontend/src/components/NavigationBar.js]: Can I please confirm why are we using bootstrap here? and did you also include material UI as a dependency?

File: frontend/src/pages/StudyRegisterationForm.js [hunk 3]
```diff
@@ -0,0 +1,1148 @@
...
+				<div style={{width: "100%", textAlign: "center", minWidth:"800px"}}>
+					<table style={{textAlign: "left", width: "100%", justifyContent: 'center', tableLayout: "fixed"}}>
+						<colgroup>
+							<col style={{width: "30%", aignItems: "left", verticalAlign: "top", textAlign: "left"}}/>
+							<col style={{width: "70%", alignItems: "right", verticalAlign: "middle", textAlign: "right"}}/>
+						</colgroup>
+						<tbody>
+							<tr>
+								<td><label style={{color:"red"}}>* </label><label>Study title:</label></td>
+								<td><input id="Study title/text" style={this.styles.input} type="text"/></td>
```
Comment (shiyuchen6688) [inline frontend/src/pages/StudyRegisterationForm.js]: I don't think we can have space in HTML element's ids, or at least it's not a good practive

**Review by shiyuchen6688 — CHANGES_REQUESTED** [review]
1. id of HTML elements should not contain space
2. is there a reason why you introduced yarn instead of keeping using npm?
3. we were using semantic UI, is there a reason for us to add bootstrap and material UI now?

File: frontend/yarn.lock (hunk 1 above)
Comment (mandana-mazaheri) [reply frontend/yarn.lock:5]: I don't use yarn and I didn't add it to my codings but as you can see, https://github.com/shiyuchen6688/V-Brain/blob/main/frontend/yarn.lock , this is used in the main branch as well, it was there from the beginning

File: frontend/src/components/NavigationBar.js (hunk 2 above)
Comment (mandana-mazaheri) [reply frontend/src/components/NavigationBar.js]: bootstrap is not used, I added that in the developing process to test something. But I don't remember if I have add material UI in dependencies. BTW, I removed them from dependencies and will rais another PR

File: frontend/src/pages/StudyRegisterationForm.js (hunk 3 above)
Comment (mandana-mazaheri) [reply frontend/src/pages/StudyRegisterationForm.js]: It's ok to use as I know, but I agree that is a bad choice, will change them in next PR

**Review by shiyuchen6688 — APPROVED** [review]

**Pull request merged**